### ISSUE TITLE
Build on go 1.12 fails

### ISSUE THREAD

**Issue by krombel** [matrix-org/matrix-federation-tester#68]
```
$ go version
go version go1.12 linux/amd64
$ go get -u github.com/constabulary/gb/...
$ gb build
golang.org/x/sys/unix.ioctl: relocation target golang.org/x/sys/unix.Syscall not defined for ABIInternal (but is defined for ABI0)
golang.org/x/sys/unix.munmap: relocation target golang.org/x/sys/unix.Syscall not defined for ABIInternal (but is defined for ABI0)
golang.org/x/sys/unix.mmap: relocation target golang.org/x/sys/unix.Syscall6 not defined for ABIInternal (but is defined for ABI0)
# github.com/matrix-org/matrix-federation-tester
github.com/matrix-org/matrix-federation-tester
FATAL: command "build" failed: exit status 2

**Comment by anoadragon453** [comment]
Also seeing this problem. Seems to be referenced in this go1.12 documentation: https://github.com/golang/proposal/blob/master/design/27539-internal-abi.md#compatibility

This bug might also contain some information: https://github.com/golang/go/issues/27539

Updating all dependencies to their latest versions did not help.

**Comment by babolivier** [comment]
I wonder if that's something we could fix with #46

**Comment by babolivier** [comment]
Fixed in #46
